catroid/src/main/java/org/catrobat/catroid/ui/fragment/UserBrickElementEditorFragment.java

```diff
@@ -278,7 +279,8 @@ public void click(int id) {
 		if (element != null && !element.isLineBreak()) {
 			int title = element.isVariable() ? R.string.edit_variable : R.string.edit_text;
 			int defaultText = element.isVariable() ? R.string.variable_hint : R.string.text_hint;
-			editElementDialog(element.getText(), true, title, defaultText);
+			boolean isFunctionName = id == 0;
```

Comment:
0 can be added as static variable in DragAndDropBrickLayoutListener, and changed at every use